src/styles/main.scss

```diff
@@ -1,3 +1,76 @@
+$cell-size: 100px;
+$count-days: 31;
+$days-order: (mon, tue, wed, thu, fri, sat, sun);
+$gap: 1px;
+
+@mixin center-content {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
 body {
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  gap: $gap;
+
+  max-width: calc(#{$cell-size} * 7 + 1px * 7 + 10px * 2);
```

Comment:
in scss you can do calculation without calc

$cell-size * 7 + $gap * 7 + 10px * 2;

for 7 and 10px you can also create variables